### ISSUE TITLE
Computing outputs for data fraction 1...FAILED: CUBLAS matrix multiplication failed

### ISSUE THREAD

**Issue by RaniemAR** [karpathy/neuraltalk#33]
When I am calling the code I am getting this error:

Outputs will be scaled by mean and standard deviation specified in NC file.
Computing outputs for data fraction 1...FAILED: CUBLAS matrix multiplication failed

it happens after creating the neural network is done and this error message shows after :

Creating the neural network... done.
Layers:
(0) input [size: 4096]
(1) lstm [size: 512, bias: 1.0, weights: 9440768]
(2) softmax [size: 8148, bias: 1.0, weights: 4179924]
(3) multiclass_classification [size: 8148]
Total weights: 13620692

Any help would be appreciated ..
Thank you very much..

**Comment by phiresky** [comment]
Yf
Am 28.10.2015 15:11 schrieb "RaniemAR" <notifications@github.com>:
>
> When I am calling the code I am getting this error:
>
> Outputs will be scaled by mean and standard deviation specified in NC
file.
> Computing outputs for data fraction 1...FAILED: CUBLAS matrix
multiplication failed
>
> it happens after creating the neural network is done and this error
message shows after :
>
> Creating the neural network... done.
> Layers:
> (0) input size: 4096 lstm size: 512, bias: 1.0, weights: 9440768 softmax
size: 8148, bias: 1.0, weights: 4179924 multiclass_classification [size:
8148]
